### ISSUE TITLE
`ember g ember-cli-focus-input` fails

### ISSUE THREAD

**Issue by adamesque** [williamsbdev/ember-cli-focus-input#4]
After installing the addon, running the generator fails:

```
⑆ ember g ember-cli-focus-input
version: 0.2.3
Unknown blueprint: ember-cli-focus-input
```

I don't know much about how addons work, but I don't see a "blueprints" directory in the source.

**Comment by williamsbdev** [comment]
You don't need to and shouldn't run the command `ember g ember-cli-focus-input`. What is that command doing:

1. `ember g` is short for `ember generate`
2. This will say, "Hey, I know how to create something from a blueprint that is shipped with ember-cli or a blueprint that has been added by an addon"
3. It will execute the blueprint and output file(s) in the appropriate places.

If you have added the below to your package.json in the `dependencies` and done an `npm install`, you should be ready to roll!

`"ember-cli-focus-input": "0.4.0"`

You can use the addon by using the component in your template like this:

`{{focus-input value=model.name class="focused-input yoyo"}}`

**Comment by williamsbdev** [comment]
Thanks for finding the bug! I merged your PR and bumped the version to 0.4.1 so I could do an `npm publish` to update the README.md on npmjs.org.

**Comment by adamesque** [comment]
Awesome. Glad I could help!
